pkg/controller/resources/network_policy.go

```diff
@@ -0,0 +1,36 @@
+package resources
+
+import (
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	networkingv1alpha1 "knative.dev/serving/pkg/apis/networking/v1alpha1"
+)
+
+const (
+	NetworkPolicyAllowAllName = "knative-serving-allow-all"
+)
+
+// MakeNetworkPolicyAllowAll creates a Kubernetes NetworkPolicy
+// allowing all traffic into the namespace of a Knative Ingress
+func MakeNetworkPolicyAllowAll(ci networkingv1alpha1.IngressAccessor) *networkingv1.NetworkPolicy {
```

Comment:
I'd just pass in the namespace instead of the whole Accessor. It's the only thing needed anyway?